packages/horizontal-layout/test/horizontal-layout.common.js

```diff
@@ -110,4 +110,52 @@ describe('vaadin-horizontal-layout', () => {
       expect(wrapper.scrollWidth).to.equal(200);
     });
   });
+
+  describe('layout improvements disabled', () => {
+    let layout, children;
+
+    describe('flex', () => {
+      beforeEach(async () => {
+        layout = fixtureSync(`
+        <vaadin-horizontal-layout>
+          <div></div>
+          <div data-full-width></div>
+        </vaadin-horizontal-layout>
+      `);
+        children = Array.from(layout.querySelectorAll('*'));
```

Comment:
nit: could use `layout.children`.

```suggestion
      children = Array.from(layout.children);
```